alica_engine/include/engine/logging/LoggingUtil.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "engine/logging/AlicaLogger.h"
+namespace alica
+{
+namespace Logging
+{
+class LogEntry
+{
+public:
+    LogEntry(Verbosity v);
+    LogEntry(LogEntry&& entry);
```

Comment:
can be defaulted I think

Reply:
even better, can delete the move constructor since we don't want to copy/move objects of this class anyway